<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>keshihua</class>
 <widget class="QWidget" name="keshihua">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1214</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>611</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>接口区</string>
   </property>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>23</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>函数</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>51</y>
      <width>24</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>参数</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_15">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>20</y>
      <width>75</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>调用</string>
    </property>
   </widget>
   <widget class="QComboBox" name="apiList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>471</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="args">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>50</y>
      <width>471</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>格式：x,x...,x;无调用参数时为空</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>611</width>
     <height>491</height>
    </rect>
   </property>
   <property name="title">
    <string>调试区</string>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>581</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QComboBox" name="comboBox_11">
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>List</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QTextBrowser" name="textBrowser"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_17">
       <property name="text">
        <string>运行</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>comboBox_11</zorder>
   <zorder>pushButton_17</zorder>
   <zorder>textBrowser</zorder>
   <zorder>groupBox</zorder>
  </widget>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>0</y>
     <width>591</width>
     <height>601</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>611</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>返回主菜单</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_16">
      <property name="text">
       <string>练习模式</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
